packages/core/src/components/button/button.css

```diff
@@ -162,32 +155,33 @@
   width: var(--size);
 }
 
-.hds-button:focus-visible:after {
-  --size: calc(100% + calc(var(--outline-width) * 2 + var(--border-width) * 2 + var(--outline-gutter) * 2));
-
-  border-color: var(--focus-outline-color);
-}
-
+.hds-button:focus-visible:after,
 .hds-button:active:after {
   --size: calc(100% + calc(var(--outline-width) * 2 + var(--border-width) * 2 + var(--outline-gutter) * 2));
 
   border-color: var(--focus-outline-color);
 }
 
 input[type="submit"].hds-button,
+input[type="reset"].hds-button,
+input[type="button"].hds-button,
 .hds-button__label {
   font-weight: inherit;
   line-height: 1.25em;
   padding: var(--spacing-s);
 }
 
-input[type="submit"].hds-button {
+input[type="submit"].hds-button,
+input[type="reset"].hds-button,
+input[type="button"].hds-button {
```

Comment:
Quite odd that these selectors weren't included when it seems like they should've been. Adding these now might cause some changes for users but I think this is more of a fix than a change.